src/shadowbox/package.json

```diff
@@ -23,7 +23,7 @@
   },
   "devDependencies": {
     "@types/js-yaml": "^3.11.2",
-    "@types/node": "^8",
+    "@types/node": "^12",
```

Comment:
If we are upgrading node, we might as well upgrade to the current LTS version 14.

Reply:
Jon originally upgraded to Node 14, but I asked him to keep it at 12 because Debian LTS (and hence my linux workstation) is still stuck on Node 12.

Reply:
We're not upgrading node.  This makes the typescript types consistent with the version of node we currently run.